kythe/web/site/getting-started.md

```diff
@@ -61,7 +62,7 @@ apt-get update
 
 apt-get install \
     asciidoc asciidoctor source-highlight graphviz \
-    gcc libssl-dev uuid-dev libncurses-dev libcurl4-openssl-dev flex clang-3.5 bison \
+    gcc libssl-dev uuid-dev libncurses-dev libcurl4-openssl-dev flex clang-3.6 libstdc++-8-dev bison \
```

Comment:
Is this generally required, or just for the internal messed up version of linux?

Reply:
Almost certainly just internal. Probably doesn't hurt ... but reduced scope of this PR to just the clang version bump.